src/main/resources/com/puppycrawl/tools/checkstyle/checks/coding/messages.properties

```diff
@@ -28,6 +28,7 @@ illegal.type=Usage of type ''{0}'' is not allowed.
 inline.conditional.avoid=Avoid inline conditionals.
 instantiation.avoid=Instantiation of {0} should be avoided.
 magic.number=''{0}'' is a magic number.
+matchxpath.match=Node is not allowed by Xpath query
```

Comment:
this message is not good.
We should find some wording to hide such AST and Xpath details of inner implementation.

So we should ADVICE users to always write custom message in config.
Please add example of it in xdoc.

but we still need to have default message. What about `Illegal code structure detected` ?

Reply:
makes sense, done